### DISCUSSION TITLE
N-runner actual

### DISCUSSION THREAD

**Discussion by marioaag** [avocado-framework/avocado#4495] [Q&A] [answered]
Is there a place where I can find the list of the features already implemented for N-runner and the upcoming ones and its possible release date?

I'm actively using Avocado and really interested in the parallel execution functionality, also willing to help if is possible and needed to get the key features in place to use them in my project

**Comment by beraldoleal** [comment, marked as answer]
@marioaag we are working to improve our documentation on that front. In fact, this is one of our goals this quarter.

For now, you can find details about the next runner here:

https://avocado-framework.readthedocs.io/en/latest/guides/contributor/chapters/runners.html

Also, do you mind sharing where you are using Avocado?

**Reply by marioaag** [reply]
Excellent I will be waiting for that... and thanks I already review that page and also this one: 
https://avocado-framework.readthedocs.io/en/latest/blueprints/BP003.html#implementation-example

I'm using it as part of test and configuration tool for IBM project doing validations for interoperability
